app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -41,9 +41,11 @@ This can help when you have (one or more of the following):
 
 ## How to do an SKE course
 
-When you apply for teacher training in England, talk to your training provider if you think you need to improve your subject knowledge. They’ll help you decide if you need to complete an SKE course, and may make it a condition of their offer to train with them.
+If you think you need to improve your subject knowledge then talk to your training provider when you apply for teacher training in England. They’ll help you to decide if you'll benefit from a SKE course. 
```

Comment:
```suggestion
If you think you need to improve your subject knowledge then talk to your training provider when you apply for teacher training in England. They’ll help you to decide if you'll benefit from an SKE course. 
```
UR showed that candidates spell out 'SKE' when saying it (es-kay-ee) so we should be using 'an' when talking about it (pronouncing it 'ski' is just what we do internally)